python/semantic_kernel/connectors/memory/qdrant/qdrant_memory_store.py

```diff
@@ -0,0 +1,385 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+"""
+QdrantMemoryStore provides functionality to add semantic memory and ANN search to Semantic Kernel.
+The QdrantMemoryStore inherits from memory_store_base for persisting and retrieving data from a Qdrant Vector Database Implementation.
+"""
+
+from qdrant_client import QdrantClient
```

Comment:
please add qdrant_client to it's own poetry install group:

`poetry add qdrant_client --group qdrant`

and then add the import in the __init__() call that throws if there's an import error.

Reply:
Addressed in https://github.com/microsoft/semantic-kernel/pull/919/commits/de7d946f07ba54e1cf9e598b7fb153d88fb3d230 and https://github.com/microsoft/semantic-kernel/pull/919/commits/556bbfcbd79638e3a10eb0baf3bb1a8a39e6f614